Story-Chief/frontend-test-stefaan-devylder
# [FEATURE] Event searching on calendar view

## Description

This PR adds the ability to search for event titles and descriptions within the calendar view.

## Closes issue(s)

N/A

## Testing Instructions

- Go to the calendar view
- Assert the new text search field with title `Search for event`
- Try to type some text and make sure the correct filtered events are there
- Test it as much as possible with many variations
- Assert the outcome in the calendar itself

## Screenshots

<img width="341" alt="Screenshot 2023-01-17 at 18 07 28" src="https://user-images.githubusercontent.com/10358801/212965127-6a5f1814-7e17-409b-bee7-29ff9e620902.png">
